### DISCUSSION TITLE
Separate PlatformScreen/platform.main into another package

### DISCUSSION THREAD

**Discussion by bilogic** [orchidsoftware/platform#1632] [Ideas]
Hi,

After playing with Orchid more, I feel that 

```BASH
# composer require orchid/platform     <-- 1. Install Orchid platform
# php artisan orchid:install           <-- 2. modify App/Model/User.php only, does not create files in App/Orchid
# composer require orchid/platform-ui  <-- 3. provide PlatformScreen/platform.main directly from package folders
# php artisan orchid:ui:install        <-- 4. create local PlatformScreen/platform.main in App/Orchid
```

This way, 

Beginner users will do 1, 2 and 3
Advanced users who want to customize PlatformScreen will do 1, 2, 3 and 4
Expert users who want to create their own UI will do step 1 and 2

**Comment by tabuna** [comment]
What will be the advantage? Except for more work for the maintainers?
For example, Laravel installs many things. There is no such thing that if we need queues or a blade, you have to install them.

**Reply by bilogic** [reply]
Hmm, when I suggested it, the idea was to separate the UI to make learning easier and also to separate code into packages to make upgrading Laravel easier. 

Anyway I think I found a solution, I will run #2 and delete the whole `App\Orchid` folder.

Thanks.
